Sources/Flowpack.Website/Classes/Aspects/GitHubCachingAspect.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
```

Comment:
No empty line before namespace, ever. Please.

Note: also applies to the other added files!